pkg/sql/logictest/testdata/logic_test/geospatial

```diff
@@ -4421,3 +4421,24 @@ POINT EMPTY                                            POINT EMPTY
 POLYGON ((-0.1 0, 1 0, 1 1, -0.1 1, -0.1 0))           POLYGON ((0 -0.1, 0 1, 1 1, 1 -0.1, 0 -0.1))
 POLYGON ((-1 0, 0 0, 0 1, -1 1, -1 0))                 POLYGON ((0 -1, 0 0, 1 0, 1 -1, 0 -1))
 POLYGON ((0 0, 1 0, 1 1, 0 1, 0 0))                    POLYGON ((0 0, 0 1, 1 1, 1 0, 0 0))
+
+query TTT
+SELECT
+  ST_AsText(a.geom) d,
+  ST_AsText(ST_Rotate(a.geom, 3.141592653589793)),
```

Comment:
i think `pi()` and `pi()/4` may be easier to read here.

Reply:
fixed in recent commit.